scripts/setup.sh

```diff
@@ -1,15 +1,17 @@
 #!/bin/bash
 
+# On first start after build
 if [ ! "$(ls -A /data)" ]; then
+    # Move over configuration files
+    mv /tmp/conf/* /data/
+    rm -rf /tmp/conf
```

Comment:
Actually nvm this wont work due to our persistent volumes.
Instead we should check for the existance of the /tmp/conf directory and act upon that.

the problem with mv is that it cannot rewrite target directories if they are not empty so for this reason we will have to move over the files individually

Reply:
Fixed